libs/hdf-converters/sample_jsons/xccdf_results_mapper/xccdf-openscap-ComplianceAsCode-ubuntu1804-hdf.json

```diff
@@ -32860,13 +32776,16 @@
                 "text": "SRG-OS-000059-VMM-000280",
                 "href": ""
               },
+              {
```

Comment:
example of a validate duplicate showing up in the hdf file cause the original sample file has it duplicated as well for some reason

![image](https://github.com/mitre/heimdall2/assets/2015821/b6fef69e-2625-4d78-ae01-1462f9e0da97)